From 4f74881e4fa0f1ca3ecd6c3be2fb2f576a564bb9 Mon Sep 17 00:00:00 2001
From: Eli Bradley <elibradley@utexas.edu>
Date: Tue, 19 Oct 2021 22:21:00 -0500
Subject: [PATCH] Fix building on Mac

---
 .gitignore     | 2 ++
 CMakeLists.txt | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/.gitignore b/.gitignore
index ddaa784..879d694 100644
--- a/.gitignore
+++ b/.gitignore
@@ -4,3 +4,5 @@ android-libs/launcher/src/com/beloko/idtech/R.java
 /*build*/
 /results/
 /.idea/
+
+.DS_Store
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c5d570..a0595c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.5.0)
 project(ECWolf)
 
+SET(CMAKE_EXE_LINKER_FLAGS "-framework Cocoa")
+
 if(POLICY CMP0054)
 	cmake_policy(SET CMP0054 NEW)
 endif()
-- 
2.33.0

