From 6967cdc016dfe2a33058a4b4b5242fd12e65bd2a Mon Sep 17 00:00:00 2001
From: Eli Bradley <elibradley@utexas.edu>
Date: Thu, 21 Oct 2021 18:40:01 -0500
Subject: [PATCH] Real fix moment

---
 .gitignore         | 2 ++
 neo/CMakeLists.txt | 6 +++++-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index bebbac75..5abe494a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -45,3 +45,5 @@ neo/.kdev4
 neo/neo.kdev4
 neo/.idea/
 neo/cmake-build-*/
+
+.DS_Store
diff --git a/neo/CMakeLists.txt b/neo/CMakeLists.txt
index 98de8c0f..f292f5ef 100644
--- a/neo/CMakeLists.txt
+++ b/neo/CMakeLists.txt
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.2)
 
 project(RBDoom3BFG)
 
+SET(OPENALDIR "/usr/local/opt/openal-soft")
+
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 if(CMAKE_MAJOR_VERSION EQUAL 3 AND CMAKE_MINOR_VERSION GREATER_EQUAL 6)
@@ -1700,7 +1702,9 @@ else()
 			${SDL_INCLUDES} ${SDL_SOURCES})
 			
 		if(OPENAL)
-			find_package(OpenAL REQUIRED)
+			# find_package(OpenAL REQUIRED)
+			SET(OPENAL_INCLUDE_DIR "/usr/local/opt/openal-soft/include")
+			SET(OPENAL_LIBRARY "/usr/local/opt/openal-soft/lib/libopenal.dylib")
 			add_definitions(-DUSE_OPENAL)
    
             # SRS - Added support for OpenAL Soft headers on OSX (vs default macOS SDK headers)
-- 
2.33.0

